pages/index.css

```diff
@@ -2,3 +2,14 @@
 @import url(../vendor/fonts/fonts.css);
 @import url(../blocks/page/page.css);
 @import url(../blocks/page/__container/page__container.css);
+@import url(../blocks/contacts/contacts.css);
```

Comment:
Let's organize our `index.css` for better clarity. Please separate each block with one empty line and add a comment before each one.

You can refer to how Shraddha implemented this [here](https://github.com/5hraddha/p-foundation-pink/blob/main/pages/index.css).
Ignore lines from 37 to 54 for now.

Put your `@import url(../blocks/visually-hidden/visually-hidden.css);` to the very bottom of the file. You can comment on it as 

```css
/* Visually Hidden Mix */
```

Reply:
@naumch1k I did it

Reply:
@SergeyAlexandrowich please move visually-hidden mix to the very bottom of `index.css` file. 

As you know browser import and read files going from top to bottom. When you place visually-hidden mix above other import, you risk to overwrite it's CSS properties if the element will have the same properties set to something else..

Reply:
@naumch1k  I did it